lms/djangoapps/grades/models.py

```diff
@@ -679,6 +680,9 @@ def _cache_key(cls, course_id):
     def _emit_grade_calculated_event(grade):
         events.course_grade_calculated(grade)
 
+    @staticmethod
+    def _emit_persistent_grade_summary_changed_event(course_id, user_id, grade):
+        events.persistent_grade_summary_changed(course_id, user_id, grade)
```

Comment:
Could we put **openedx** to the name of the function? in this way, we can differentiate between tasks that are linked to internal processes of the platform and procedures that are associated with external processes (in this case openedx-events). I propose that it looks like this **openedx_persistent_grade_summary_changed**

Reply:
Yes, I think so, I have already made the change, thank you for your advice.